modules/ocf_desktop/files/skel/config/brave/session-store-1

```diff
@@ -0,0 +1,15 @@
+{
+    "settings": {
+      "general.check-default-on-startup": false,
+      "general.is-default-browser": false,
+      "general.homepage": "https://www.ocf.berkeley.edu/about/lab/open-source",
```

Comment:
Get this information [from hiera](https://github.com/ocf/puppet/blob/master/hieradata/common.yaml#L23) instead?

Reply:
This means that this file would need to be a template, but you can do something like this:

https://github.com/ocf/puppet/blob/master/modules/ocf/templates/firefox/prefs.js.erb#L1

Reply:
Yeah, good idea. In that case would it be better to put the `file` declaration in [brave.pp](https://github.com/ocf/puppet/blob/master/modules/ocf/manifests/packages/brave.pp)?

```  
file {
    '/etc/skel/.config/brave/session-store-1':
      content => template('ocf/brave/session-store-1.erb'),
      require => Package['brave'];
  }
```

Reply:
Yeah, I think it probably would be better to do that, is there no system-wide config we could use at all? :/

Reply:
[I don't think so](https://github.com/brave/browser-laptop/wiki/Filesystem-Hierarchy). After a bit of poking around it looks like they load the app state [here](https://github.com/brave/browser-laptop/blob/0b9e0a8b122196554d28c2b63176fc81faef7460/app/sessionStore.js#L961). The [getStoragePath()](https://github.com/brave/browser-laptop/blob/0b9e0a8b122196554d28c2b63176fc81faef7460/app/sessionStore.js#L58-L60) uses `app.getPath()` which, according to [Electron](https://github.com/electron/electron/blob/master/docs/api/app.md#appgetpathname), points to `~/.config`. Couldn't find any method that looks for global settings first =/. No one on their Discord has gotten back to me either.

Reply:
From poking around the package installed, that doesn't add anything either, so I think just going with what you have is ok, maybe add a `TODO` about it though so that we can switch if/when a global config is added.